core/variant/variant.cpp

```diff
@@ -2036,7 +2036,7 @@ Variant::operator ::RID() const {
 		}
 #endif
 		Callable::CallError ce;
-		Variant ret = _get_obj().obj->call(CoreStringNames::get_singleton()->get_rid, nullptr, 0, ce);
+		Variant ret = _get_obj().obj->call(SNAME("get_rid, nullptr, 0, ce");
```

Comment:
Regex didn't handle commas properly.

Reply:
I know, that's why I marked it as draft :). Should be fixed now.